src/App.js

```diff
@@ -1,23 +1,28 @@
 import logo from './logo.svg';
 import './App.css';
+import ListArea from './components/ListArea/ListArea.js'
+import { ListContextProvider } from "./context";
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
       </header>
+      <ListContextProvider>
+      <div className='main-body'>
+        <div className='input-area'>
```

Comment:
split ```input-area``` and ```answer-area``` into separate components. it will make your app component more clean